CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable, unreleased changes to this project will be documented in this file.
 
 # 3.7.0 [Unreleased]
 
+### Other changes
+
+- Started process of removing lazy objects from codebase. This is the first step on the road to async.
```

Comment:
I would be more explicit here - "Replaced `context.app` with dataloader" or something like that.

Reply:
Not sure this is even worth mentioning in the changelog, as `context.app` is likely not even useful for plugin authors.

Reply:
Transparency, I guess :thinking:

Reply:
Transparency at scale is noise. 😄 We really need to make sure we only write about things that are relevant to our users as their attention is a finite resource.